carlos/rocker/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Tech Blog</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans|PT+Sans:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+
+<body>
+
+    <header class="site-header">
+        <div class="contenedor">
+            <div class="barra">
+                <a href="/">
+                    <h1 class="no-margin">Tech<span> Blog</span></h1>
+                </a>
+                <nav class="navegacion">
+                    <a href="nosotros.html">About Us</a>
+                    <a href="cursos.htmnl">Vision</a>
+                    <a href="contacto.html">Contact</a>
+                </nav>
+            </div>
+            <!--barra---->
+            <div class="texto-header">
+                <h2 class="no-margin"><span class="back">News about new techs</span></h2>
+                <p class="no-margin"><span class="back">Stay tunned!</span></p>
+            </div>
+        </div>
+        <!--contendedor-->
+    </header>
+
+    <div class="contenido-principal contenedor">
+        <main class="blog">
+            <h2>Our Blog</h2>
+            <article class="entrada-blog">
+                <div class="imagen">
+                    <img src="img/blog1.jpg" alt="Imagen blog">
+                </div>
+                <div class="contenido-blog">
+                    <h3 class="no-margin">New technologies</h3>
+                    <p>Quisque lorem ex, venenatis id mauris nec, pretium pharetra orci. Nam vehicula tristique imperdiet. Cras eu massa mi. Integer congue ullamcorper arcu, vehicula interdum est varius in. Sed eget elementum nisl, ut volutpat lorem. Nunc
+                        quis porttitor nisi, nec lacinia nunc. </p>
+                    <a href="entrada.html" class="btn btn-primario">Read more</a>
```

Comment:
Its confusing to have as class name a reference of a button on an anchor, both elements are different and have a different use. I would use [BEM](https://css-tricks.com/bem-101/) everywhere instead of this "material-ui" like coding convention. This class can be named `contenido-blog__read-more-link`